R/skip.R

```diff
@@ -109,27 +113,33 @@ skip_if_not_installed <- function(pkg, minimum_version = NULL) {
     }
   }
 
-  return(invisible(TRUE))
+  invisible()
 }
 
 #' @export
 #' @rdname skip
 skip_if_offline <- function(host = "r-project.org") {
   skip_on_cran()
   skip_if_not_installed("curl")
-  has_internet <- !is.null(curl::nslookup(host, error = FALSE))
-  if (!has_internet) {
+
+  if (!has_internet(host)) {
     skip("offline")
+  } else {
+    invisible()
   }
```

Comment:
Nitpick, but I try to avoid negating the condition when I can.
```suggestion
  if (has_internet(host)) {
    invisible()
  } else {
    skip("offline")
  }
```

Reply:
Given that we already have the `skip_if()` and `skip_if_not()` helpers that do this, I just refactored to use them in more places.